generators/app/templates/_dockerTaskGeneric.ps1

```diff
@@ -51,7 +51,13 @@ function CleanAll () {
 
 # Builds the Docker image.
 function BuildImage () {
-    $dockerFileName="Dockerfile.$Environment"
+    $dockerFileName = $Null
+    if ($Environment -eq "Release") {
+        $dockerFileName = "dockerfile"
+    }
+    else {
+        $dockerFileName = "dockerfile.$Environment"
```

Comment:
Consider integrating the if around the $taggedImageName from below into this if, seems silly to check if the environment is release twice.